models/contact_views/contactview_metadata.sql

```diff
@@ -0,0 +1,28 @@
+{{
+    config(
+        materialized = 'view',
+        indexes=[
+            {'columns': ['contact_uuid']},
+            {'columns': ['parent_uuid']},
+            {'columns': ['type']},
+            {'columns': ['uuid']},
+        ]
+    )
+}}
+
+SELECT
+    raw_contacts.doc ->> '_id'::text AS uuid,
```

Comment:
The `raw_contacts` prefix is probably not needed since we are only referencing a single table